backend/src/models/student.js

```diff
@@ -52,7 +52,7 @@ class Student extends Model {
   static get jsonSchema () {
     return {
       type: 'object',
-      required: ['PhoneNumber', 'FirstName', 'LastName', 'StatusID'],
+      required: ['PhoneNumber', 'FirstName', 'LastName'],
```

Comment:
Do we really need to delete this? We aren't saying that a Student can have no status, we are just saying that the frontend does not need to pass the status. If it doesn't, the backend will still ensure that the inserted Student has the default status.

Reply:
yes, we have put that back in. Thanks for the catch!